alexandria-reader/frontend
# Bug fixes, layout fixes with texts

Improved texts layout. 
Fixed bug where delete button opened text.
Moved Link element inside IndividualText. 
Fixed bug where text was deleted but wasnt removed without a refresh.
Updated app title, description and favicon.
Fixed bug, adding translations now appears instant to the user.

Closes #46 
Closes #50 
Closes #52 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  X  | :bug: Bug fix              |
|     | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing Steps / QA Criteria

Try deleting a text, it should be deleted from the dom instantly.
Add multiple texts, the layout should be in the style of cards.
Add a new translation, it should appear instantly.
